contracts/Market.sol

```diff
@@ -247,27 +247,22 @@ contract Market is IMarket, Ownable, ERC721 {
 		assert(payout > 0);
 
 		return _back(
-			nonce,
 			propositionId,
 			marketId,
 			wager,
-			odds,
 			close,
 			end,
-			signature
+			payout
```

Comment:
backing no longer requires a signature?

Reply:
That's just the internal `_back()` method

Reply:
ah so the signature is checked in the public `back` function then?